pep517/build-meta.py

```diff
@@ -0,0 +1,3 @@
+from . import build_meta
+
+__name__ == '__main__' and build_meta.main()
```

Comment:
wow I've never seen this cryptic way of doing this 🤔

Reply:
I agree it's idiosyncratic, but I've run it by other seasoned devs and gotten at least a nod of endorsement about the one-line approach, so I've started using it. I personally prefer it to the traditional approach for two reasons: (1) it fits on one line, and (2) it encourages exactly one expression for `__main__`. What I'd really prefer is a builtin that does what [autocommand](https://pypi.org/project/autocommand) does - decorate a function to be run as main. I use that wherever I can rely on Python 3 and third-party dependencies.

Reply:
I'm -1 on this. Let's keep to the standard `if __name__ == '__main__'` idiom. Why make it harder for future maintainers to understand for little benefit?

Reply:
I definitely find it hard to read, but that's only me perhaps.

Reply:
I updated this and ultimately removed the file entirely after I found that `pep517.meta` was a more suitable name for both the command and the API module.